packages/desktop-client/src/components/sidebar/Accounts.tsx

```
@@ -43,126 +39,149 @@ export function Accounts({
+        )}
 
-      {showClosedAccounts &&
-        closedAccounts.map(account => (
+        {offbudgetAccounts.map((account, i) => (
           <Account
             key={account.id}
             name={account.name}
             account={account}
+            connected={!!account.bank}
+            pending={syncingAccountIds.includes(account.id)}
+            failed={failedAccounts && failedAccounts.has(account.id)}
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)